docs/src/pages/guides/FHIROperationFramework.md

```diff
@@ -0,0 +1,348 @@
+---
+layout: default
+title: IBM FHIR Server - Extended Operations Framework
+date:  2020-09-29
+---
+
+# IBM FHIR Server - Extended Operations Framework
+
+The HL7 FHIR Specification describes a [light weight framework](https://www.hl7.org/fhir/r4/operations.html) of extended operations for the REST API.
```

Comment:
I noticed the spec itself calls this "lightweight" but IMHO we should remove such value statements from the technical docs (similar to avoiding terms like "simply do X" or "merely do Y").

Reply:
sure